[BUG] Compile issue since commit 7bfecf6

### Did you test the latest `bugfix-2.1.x` code?

Yes, and the problem still exists.

### Bug Description

With `DGUS_LCD_UI_MKS` defined, the file `Marlin/src/lcd/extui/dgus/DGUSScreenHandler.cpp` no longer compiles with the following error:
`Marlin\src\lcd\extui\dgus\DGUSScreenHandler.cpp:329:7: error: 'sendInfoScreenMKS' was not declared in this scope; did you mean 'sendInfoScreen_P'?`

The issue is that `sendInfoScreenMKS` is not defined in the `DGUSScreenHandler` class, only in the child class `DGUSScreenHandlerMKS`. The fix here is to either reintroduce the deleted `screen.` or prefix it with `DGUSScreenHandlerClass::`

So [line 329](https://github.com/MarlinFirmware/Marlin/blob/db4990f2c4f36eb56323c262b42042ec8efb6ad9/Marlin/src/lcd/extui/dgus/DGUSScreenHandler.cpp#L329) should become
`screen.sendInfoScreenMKS(F("NOTICE"), nullptr, F("SD card error"), nullptr, mks_language_index);`
OR
`DGUSScreenHandlerClass::sendInfoScreenMKS(F("NOTICE"), nullptr, F("SD card error"), nullptr, mks_language_index);`

Both versions compile with no issues.

### Bug Timeline

Since commit 7bfecf6

### Expected behavior

_No response_

### Actual behavior

_No response_

### Steps to Reproduce

_No response_

### Version of Marlin Firmware

latest bufgix-2.1.x

### Printer model

_No response_

### Electronics

_No response_

### LCD/Controller

_No response_

### Other add-ons

_No response_

### Bed Leveling

None

### Your Slicer

None

### Host Software

None

### Don't forget to include

- [x] A ZIP file containing your `Configuration.h` and `Configuration_adv.h`.

### Additional information & file uploads

_No response_

--- a/Marlin/src/lcd/extui/dgus/DGUSScreenHandler.cpp
+++ b/Marlin/src/lcd/extui/dgus/DGUSScreenHandler.cpp
@@ -326,7 +326,7 @@ void DGUSScreenHandler::sendHeaterStatusToDisplay(DGUS_VP_Variable &var) {
   void DGUSScreenHandler::sdCardError() {
     DGUSScreenHandler::sdCardRemoved();
     #if DGUS_LCD_UI_MKS
-      sendInfoScreenMKS(F("NOTICE"), nullptr, F("SD card error"), nullptr, mks_language_index);
+      screen.sendInfoScreenMKS(F("NOTICE"), nullptr, F("SD card error"), nullptr, mks_language_index);
     #else
       sendInfoScreen(F("NOTICE"), nullptr, F("SD card error"), nullptr);
     #endif

